packages/api/endpoints/rules.js

```diff
@@ -99,6 +106,8 @@ async function put(req, res) {
   }
 
   const d = await model.update(originalData, data);
+  req.rulesRecord = d;
```

Comment:
Can we put this inside the `inTestMode()` block, as with the collections endpoint?

Reply:
This is changed with the removal of the extra middleware.